public/index.html

```diff
@@ -64,12 +64,18 @@
   <body>
     <main class="container">
       <h1 class="center-text">Node Notification Service</h1>
-      <h2 class="center-text">Push Notification dispatcher of the Schul-Cloud project.</h2>
+      <h2 class="center-text">A Push Notification dispatcher</h2>
 
-      <h3 class="center-text"><a href="sent.html"> Send Interface</a> </h3>
+      <p class="center-text">Interface</p>
+      <h3 class="center-text"><a href="sent.html">Send Interface</a></h3>
+      <h3 class="center-text"><a href="docs">API Documentation</a></h3>
+
+      <p class="center-text">Clients</p>
+      <h3 class="center-text"><a href="firebaseTest.html">Chrome, Firefox, Opera (firebase)</a></h3>
+      <h3 class="center-text"><a href="apnTest.html">Safari (apn)</a></h3>
 
       <footer>
-        <p class="center-text">For more information on Schul-Cloud see <a href="https://schul-cloud.org/" alt="Schul-Cloud" target="blank">schul-cloud.org</a>.</p>
+        <p class="center-text">For more information on the project see the public <a href="https://github.com/schulcloud/node-notification-service" alt="Code on GitHub" target="blank">GitHub page</a>.</p>
```

Comment:
CSS styling is a little off here.
![image](https://cloud.githubusercontent.com/assets/7360440/22389140/70dbd34c-e4e4-11e6-998e-0dff7c292a43.png)